src/main/java/training/commands/WristCommands.java

```diff
@@ -0,0 +1,12 @@
+package training.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import training.subsystems.Wrist.WristSubsystem;
+import utils.WristDirection;
+
+public class WristCommands {
+    private final static WristSubsystem wristSubsystem = WristSubsystem.getInstance();
+    public Command MoveUpWristCommand = new InstantCommand(() -> wristSubsystem.rotate(WristDirection.kUp), wristSubsystem);
+    public Command MoveDownWristCommand = new InstantCommand(() -> wristSubsystem.rotate(WristDirection.kDown), wristSubsystem);
+}
```

Comment:
make the two command functions and not var

u know what, there is functional command that gets functions to run in every part of the command, maybe make the elbow command functions also like in same file,

Reply:
So the problem is that it does not inherit from `GBSubsystem` and hence doesn't show in the DashBoard the running Command? Wouldn't it be better to implement GBInstantCommand instead?

Reply:
there is no problem with it being a  variable in this case, but it makes the command like singleton so functions is generally preferred in most of command cases, About you ques the wrist subsystem is the one that needs to inheritance from GBSubsystem and not the command

Reply:
Oh I get it now, maybe it's a bad idea to read CRs at 1:30AM

Reply:
Maybe😉, and maybe it’s a good idea to sleep🤔